lib/Parse/ParseDecl.cpp

```diff
@@ -5048,13 +5048,13 @@ ParserStatus Parser::parseTypeAttribute(TypeOrCustomAttr &result,
 
 static std::optional<LifetimeDependenceKind>
 getLifetimeDependenceKind(const Token &T) {
-  if (T.isContextualKeyword("_copy")) {
+  if (T.isContextualKeyword("_copy") || T.isContextualKeyword("_inherit")) {
     return LifetimeDependenceKind::Copy;
   }
   if (T.isContextualKeyword("_consume")) {
     return LifetimeDependenceKind::Consume;
   }
-  if (T.isContextualKeyword("_borrow")) {
+  if (T.isContextualKeyword("_borrow") || T.isContextualKeyword("_scope")) {
```

Comment:
Do we need to support `_inherit` and `_scope` in the new parser as well?

Reply:
This spelling is for SIL tests only. I don't think this code path goes through the new parser.

Reply:
Ah, I didn’t read this properly. Yes, looks like SIL-only which doesn’t need support in the new parser.